cmd/nexusUpload.go

```diff
@@ -414,28 +417,31 @@ func addMavenTargetArtifact(utils nexusUploadUtils, uploader nexus.Uploader,
 	if packaging == "" {
 		packaging = "jar"
 	}
-	filePath := composeFilePath(targetFolder, finalBuildName, packaging)
-	return addArtifact(utils, uploader, filePath, "", packaging)
-}
 
-func addMavenTargetSubArtifacts(utils nexusUploadUtils, uploader nexus.Uploader,
-	additionalClassifiers, targetFolder, finalBuildName string) error {
-	if additionalClassifiers == "" {
-		return nil
-	}
-	classifiers, err := getClassifiers(additionalClassifiers)
-	if err != nil {
-		return err
-	}
-	for _, classifier := range classifiers {
-		if classifier.Classifier == "" || classifier.FileType == "" {
-			return fmt.Errorf("invalid additional classifier description (classifier: '%s', type: '%s')",
-				classifier.Classifier, classifier.FileType)
-		}
-		filePath := composeFilePath(targetFolder, finalBuildName+"-"+classifier.Classifier, classifier.FileType)
-		err = addArtifact(utils, uploader, filePath, classifier.Classifier, classifier.FileType)
-		if err != nil {
-			return err
+	fileTypes := []string{packaging}
+	if packaging != "jar" {
+		fileTypes = append(fileTypes, "jar")
+	}
+
+	for _, fileType := range fileTypes {
+		pattern := targetFolder + "/*." + fileType
+		matches, _ := utils.glob(pattern)
```

Comment:
Why ignoring the error?

Reply:
Yes, this is a deliberately ignored, since it is documented to fail only for malformed patterns. It will not return file system errors or anything like that.